.coafile

```diff
@@ -31,13 +31,13 @@ max_line_length = 120
 [all.links]
 bears = InvalidLinkBear
 
-[html]
+[all.html]
 bears = HTMLLintBear
 files = **.html
 htmllint_ignore = optional_tag, concerns_separation
 ignore = public/**
 
-[javascript]
+[all.javascript]
```

Comment:
preferred quotation in section `all` is `'`, maybe that will affect JS's usage of `"`